.readthedocs.yaml

```diff
@@ -0,0 +1,17 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  # Build the Ansible docsite
+  commands:
+    - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -m pip install --exists-action=w -r requirements.in -c requirements.txt
+    - "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -I build.py
+    - mkdir -pv "${READTHEDOCS_OUTPUT}"
+    - cp -rv output "${READTHEDOCS_OUTPUT}"/html
```

Comment:
@webknjaz I'm running into a problem with that `>-` block with the `"${READTHEDOCS_VIRTUALENV_PATH}"` variable. Have you got an example of this in another repo somewhere?

Reply:
Could you describe the problem, since RTD isn't enabled and I can't see the logs? I was typing this in from phone and might've made typos. In general, I use `>-` so that it's unnecessary to use quotes, plus it concatenates the entire node as one space-separated line.

Reply:
@oraNod is it possible that the problem is actually in the next instruction where I didn't use `>-`? I expect that having `"` at the beginning may confuse the YAML parser. Try applying `>-` there too?